Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -1240,7 +1248,25 @@ public static string GetXMLGingerVersion(string xml, string xmlFilePath)
             }
         }
 
+        private static bool HandleClassToSkip(XmlReader xdr, string className)
+        {
+            //TODO: If need to add more classed then create a dictionary of Classes to Skip
+            if (className == "GingerCore.DataSource.ActDSConditon")
+            {
+                int level = xdr.Depth;
```

Comment:
Create a function to move xdr to next element, so in case needed can be used in other class case